File: market/test/java/ru/yandex/market/logistics/management/AbstractTest.java
package ru.yandex.market.logistics.management;

import org.assertj.core.api.JUnitJupiterSoftAssertions;
import org.junit.jupiter.api.extension.RegisterExtension;

import static org.junit.jupiter.params.ParameterizedTest.INDEX_PLACEHOLDER;

public abstract class AbstractTest {
    protected static final String TUPLE_PARAMETERIZED_DISPLAY_NAME = "[" + INDEX_PLACEHOLDER + "] {0}";

    @RegisterExtension
    protected final JUnitJupiterSoftAssertions softly = new JUnitJupiterSoftAssertions();
}

File: market/test/kotlin/ru/yandex/market/logistics/mqm/AbstractTest.kt
package ru.yandex.market.logistics.mqm

import org.junit.jupiter.params.ParameterizedTest
import ru.yandex.market.common.util.DateTimeUtils
import java.time.Instant
import java.time.LocalDateTime

open class AbstractTest {
    companion object {
        const val TUPLE_PARAMETERIZED_DISPLAY_NAME = "[" + ParameterizedTest.INDEX_PLACEHOLDER + "] {0}"
    }

    protected fun String.toInstant(): Instant =
        LocalDateTime.parse(this)
            .atZone(DateTimeUtils.MOSCOW_ZONE)
            .toInstant()
}
